src/main/resources/static/signin.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="pl">
+	<head> 
+       <meta charset="utf-8">
+       <title>Sign In</title>
+       <meta name="description" content="same barachło">
+       <link rel="text/plain" rel="autor" href="humans.txt">
+       <link rel="stylesheet" href="styl1.css">
```

Comment:
styl1.css will not be loaded, becose it's in css directory, change to href="css/styl1.css", also rename it to style.css or other english name.

This is hot I see this page
![image](https://user-images.githubusercontent.com/10707837/36495513-c617497a-1735-11e8-8237-56f9abebcc91.png)